src/test/java/com/iexec/core/worker/WorkerServiceTests.java

```diff
@@ -512,27 +623,34 @@ void shouldGetTotalAliveCpu() {
         Gauge aliveTotalCpuGauge = Metrics.globalRegistry.find(WorkerService.METRIC_CPU_TOTAL_GAUGE).gauge();
         Gauge aliveAvailableCpuGauge = Metrics.globalRegistry.find(WorkerService.METRIC_CPU_AVAILABLE_GAUGE).gauge();
 
-        Assertions.assertThat(aliveWorkersGauge).isNotNull();
-        Assertions.assertThat(aliveTotalCpuGauge).isNotNull();
-        Assertions.assertThat(aliveAvailableCpuGauge).isNotNull();
+        assertThat(aliveWorkersGauge).isNotNull();
```

Comment:
with the test modification, you didn't take the opportunity to use assertAll ?

Reply:
Please see aeb00e1